SuperGoodSoft/solidus_taxjar
# Support backfilling for specific date range

How do you manually test these changes? (if applicable)
---

- Create some orders for a variety of date ranges
- Backfill the order for a specific date range
- Verify only the appropriate orders are backfilled

Merge Checklist
---


Screenshots
---
